client/src/components/App.jsx

```diff
@@ -48,11 +50,11 @@ class App extends React.Component {
   }
 
   render() {
-    const { headline, listings } = this.state;
+    const { headline, listings, city } = this.state;
     return (
       <div>
         <div>{headline}</div>
```

Comment:
@AnimalFace Putting headline in a stylized component rather than just a div may make it easier to style.